Src/AutoFixture.NUnit3.UnitTest/ThrowingStubFixture.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    /// <summary>
+    /// A <see cref="IFixture"/> for the benefit of creating stubs of <see cref="AutoDataAttribute"/> for unit testing
+    /// </summary>
+    public class ThrowingStubFixture : IFixture
```

Comment:
The reason that most Test Doubles are called `DelegatingXyz` is because they _delegate_ their implementation to a (configurable) Func or Action.

AFAICT, this `ThrowingStubFixture` doesn't delegate its implementation. It only throws exceptions upon being called. Isn't the name an accurate description of that? Am I missing something?